### DISCUSSION TITLE
View LaTeX pdf in VSCode internal tab not working

### DISCUSSION THREAD

**Discussion by j7168908jx** [James-Yu/LaTeX-Workshop#3149] [Q&A]
Sorry for the inconvenience of not providing further details, but I'm now in the same situation as Issue  #2760. VS Code automatically updated Workshop to 8.19.2, and I then found the `View in VS Code tab` not working. It opened another tab but remained blank forever.

Reinstall ver 8.19.2 makes no difference. After browsing Github and reading #2760, then downgrade to 8.18.0, everything goes fine now.

I'm not sure if a fix of this bug is undergoing, as there is no related issue opening. So I'm here to report this bug again.

I'm sorry that I cannot provide further info as I'm in a hurry with my final report and will continue working on my due with 8.18.0.

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.
